control_msgs/msg/MultiInterfaceJointState.msg

```diff
@@ -0,0 +1,6 @@
+Header header
+
+# List of resource names, e.g. ["arm_joint_1", "arm_joint_2", "gripper_joint"]
+string[] resource_name
```

Comment:
it the massage has `Joint` in the name than one could use `joint` instead of `resource`. Otherwise, the message could be `MultiInterfaceResourceState`.

@Karsten1987: Would this be in-line with your comment in the today's meeting?

Reply:
joint it is for consistency for now